test/legacy/sample.html

```diff
@@ -0,0 +1,56 @@
+<html>
+<head>
+    <meta name='ocr-id' value='OCRopus Revision: 312'>
+    <meta name='ocr-recognized' value='lines text'>
+    <meta name='DC.creator' value='James T. Kirk'>
+    <meta name='DC.title' value='Nomad'>
+    <meta name='DC.publisher' value='NASA'>
+</head>
```

Comment:
Since I don't know the original intention of the `tests` folder, I kept the `legacy` folder as it was, only changed the shell script to test something. We also don't use the `dcsample.xml` `dcsample2.xml` files.